backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/window/model/Document.java

```diff
@@ -117,6 +117,8 @@ public static Builder builder(final DocumentEntityDescriptor entityDescriptor)
 	private static final ReasonSupplier REASON_Value_Refreshing = () -> "direct set on Document (refresh)";
 	private static final ReasonSupplier REASON_Value_ParentLinkUpdateOnSave = () -> "parent link update on save";
 
+	private static final String IsSOTrx_NAME = "IsSOTrx";
```

Comment:
consider using de.metas.ui.web.window.WindowConstants#FIELDNAME_IsSOTrx instead

Reply:
and remove this constant

Reply:
Would need new **circular** dependency